sippy-ng/src/component_readiness/CompReadyUtils.js

```diff
@@ -72,6 +72,8 @@ export const cancelledDataTable = {
     },
   ],
 }
+export const jiraUrlPrefix = 'https://issues.redhat.com/browse/'
```

Comment:
Optional, things like this are good candidates for the environment https://github.com/openshift/sippy/blob/main/sippy-ng/.env.development

Might be useful if you ever want to play around with the staging jira environment